src/ServiceControlInstaller.Engine/Instances/ServiceControlNewInstance.cs

```diff
@@ -37,8 +37,22 @@ public override void WriteConfigurationFile()
 
         public override string DirectoryName => "ServiceControl";
 
+        public List<RemoteInstanceSetting> RemoteInstances { get; set; } = new List<RemoteInstanceSetting>();
```

Comment:
I was thinking if it would be better to use a HashSet and more the logic for filtering into the comparer but I guess it doesn't matter really here

Reply:
This 'pattern' has been used in 2-3 different places. We can refactor it and move everything into a custom collection of sort, but considering there isn't much logic around it, I thought it'd be overkill.